alma/views/js/alma-inpage.js

```diff
@@ -24,6 +24,8 @@
 let inPage = undefined;
 let paymentButtonEvents = [];
 
+onloadAlma();
+
 window.addEventListener("load", function() {
     onloadAlma();
     window.__alma_refreshInpage = onloadAlma;
```

Comment:
tu lance 2X la fonction onLoadAlma?

Reply:
@Francois-Gomis 
It's ok in my env, I removed it
cc @ClaireAlma